### ISSUE TITLE
Connectors are lost when opened in Editor

### ISSUE THREAD

**Issue by ThomasHagenJohansen** [Microsoft/vscode-azurelogicapps#79]
I have this defined in Azure

    "$connections": {
        "value": {
            "dynamicsax": {
                "connectionId": "/subscriptions/[my_subscription]/resourceGroups/[my_resource_group]/providers/Microsoft.Web/connections/dynamicsax",
                "connectionName": "dynamicsax",
                "id": "/subscriptions/[my_subscription]/providers/Microsoft.Web/locations/westeurope/managedApis/dynamicsax"
            },
            "sql": {
                "connectionId": "/subscriptions/[my_subscription]/resourceGroups/[my_resource_group]/providers/Microsoft.Web/connections/sql-1",
                "connectionName": "sql",
                "id": "/subscriptions/[my_subscription]/providers/Microsoft.Web/locations/westeurope/managedApis/sql"
            }
        }
    }

and this is what I see in VS Code, when opened in Editor

    "parameters": {
        "$connections": {
            "defaultValue": {},
            "type": "Object"
        }
    },

When I update Azure, the Logic Apps fail to run, because the Connectors are overwritten.

**Comment by joechung-msft** [comment]
I was able to reproduce the issue.  It looks like we are no longer expanding the $connections parameter on load so saving overwrites it with the empty parameter.

**Comment by joechung-msft** [comment]
Clarifying something here.

The "parameters" object in the definition -- as seen in the editor -- should be empty.

````
{
  "properties": {
    "definition": {
      "parameters": {
        "$connections": {
          "defaultValue": {},
          "type": "object"
        }
      }
    }
  }
}
````

The "parameters" object as seen in the portal actually comes from a "parameters" property in the Logic App ARM resource "properties" object.

````
{
  "properties": {
    "parameters": {
      "$connections": {
        "value": {
          "dynamicsax": {
            "connectionId": "/subscriptions/[my_subscription]/resourceGroups/[my_resource_group]/providers/Microsoft.Web/connections/dynamicsax",
            "connectionName": "dynamicsax",
            "id": "/subscriptions/[my_subscription]/providers/Microsoft.Web/locations/westeurope/managedApis/dynamicsax"
          },
          "sql": {
            "connectionId": "/subscriptions/[my_subscription]/resourceGroups/[my_resource_group]/providers/Microsoft.Web/connections/sql-1",
            "connectionName": "sql",
            "id": "/subscriptions/[my_subscription]/providers/Microsoft.Web/locations/westeurope/managedApis/sql"
          }
        }
      }
    }
  }
}
````

**Comment by joechung-msft** [comment]
The fix will be in this week's release (0.1.13), ETA Thursday (pending designer regression testing).
